step/configure_steam_wineprefix.sh

```diff
@@ -2,12 +2,19 @@
 
 if [ -n "${game_protontricks[*]}" ]; then
 	log_info "applying protontricks ${game_protontricks[@]}"
-	WINETRICKS="$downloaded_winetricks" \
-	protontricks "$game_appid" -q "${game_protontricks[@]}" \
-		| "$dialog" loading "Configuring game prefix\nThis may take a while"
+	if [ "$using_flatpak_protontricks" == "0" ]; then
+		WINETRICKS="$downloaded_winetricks" \
+		protontricks "$game_appid" -q "${game_protontricks[@]}" \
+			| "$dialog" loading "Configuring game prefix\nThis may take a while"
+	else
+		flatpak run \
+			--env="WINETRICKS=$downloaded_winetricks" \
```

Comment:
Hi! I've been trying to install the 4.2.2 pre-release which uses this code, and I've bumped into an issue that I'm fairly certain is caused by this line. It will complain about WINETRICKS not being installed
```
protontricks (ERROR): The WINETRICKS path is invalid, please make sure Winetricks is installed in that path!
Winetricks isn't installed, please install winetricks in order to use this script!
```

This is AFAIK because `$downloaded_winetricks` isn't accessible inside the `protontricks` Flatpak.

Locally I have circumvented this issue by editing `step/download_external_resources.sh`:
```
- downloaded_winetricks="$downloads_cache/winetricks"
+ downloaded_winetricks="$HOME/.var/app/com.valvesoftware.Steam/winetricks"
```

Hopefully this is helpful to you!